e2e/test/scenarios/onboarding/search/recently-viewed.cy.spec.js

```diff
@@ -53,6 +53,20 @@ describe("search > recently viewed", () => {
 
     cy.url().should("match", /\/question\/\d+-orders$/);
   });
+
+  it("shows up-to-date list of recently viewed items after another page is visited", () => {
+    openPeopleTable();
+    cy.findByTextEnsureVisible("Address");
+
+    cy.findByPlaceholderText("Search…").click();
+    cy.findByTestId("loading-spinner").should("not.exist");
+
+    assertRecentlyViewedItem(0, "People", "Table");
+    assertRecentlyViewedItem(1, "Orders in a dashboard", "Dashboard");
+    assertRecentlyViewedItem(2, "Orders", "Question");
+
+    cy.findAllByTestId("recently-viewed-item-title").should("have.length", 3);
+  });
```

Comment:
@iethree The test we began writing in e2e/test/scenarios/onboarding/search/search.cy.spec.js wasn't working for some reason, but this similar test works

Reply:
🤔 I suspect that this test doesn't quite test what we want, and I think it would pass on master without your change - we don't do anything to assert that the recents endpoint gets called again and re-fetches updated data. 

I also think this test will be most useful on the home page, which has a recents endpoint call by default, which lets us assert that the search dropdown calls it again.

Reply:
Thank you for that kind explanation 🙏🙏🙏

The test now uses a click (on line 70) rather than `cy.visit`, so I think it now properly tests the right behavior?

The test begins by visiting the home page (on line 34) - let me know if this isn't what you were referring to, though. Also, the test now fails on `master` and works on this branch.

Let me know if I'm missing anything!